app/models/admin/delete/dogs.js

```diff
@@ -0,0 +1,110 @@
+const { breadcrumbs } = require('../../../constants/admin')
+const { formatToGds } = require('../../../lib/date-helpers')
+
+/**
+ * @param {{
+ *  column: 'status'
+ *  order: 'ASC'|'DESC'
+ * }} sort
+ * @param {string} column
+ * @return {string}
+ */
+const getAriaSort = (sort, column) => {
+  let calculatedColumn = column
+
+  if (column === undefined) {
+    calculatedColumn = 'status'
+  }
+
+  if (calculatedColumn === sort.column && sort.order === 'DESC') {
+    return 'descending'
+  }
+
+  if (calculatedColumn === sort.column) {
+    return 'ascending'
+  }
+
+  return 'none'
+}
+/**
+ * @param {{
+ *  column: 'joinedExemptionScheme'
+ *  order: 'ASC'|'DESC'
+ * }} sort
+ * @param {string} column
+ * @return {string}
+ */
+const columnLink = (sort, column) => {
```

Comment:
as above

Reply:
Haha - I'm amused you picked up on this when I copied it from your code (that doesn't have a unit test). Ooooops! I'll add something